crates/server/src/api/v1/mod.rs

```diff
@@ -90,6 +94,35 @@ pub async fn not_found() -> impl IntoResponse {
     }
 }
 
+/// An extractor for the `Warg-Registry` header.
```

Comment:
Can we extend this doc comment to mention that it currently returns an error if it's ever provided currently?

Reply:
Yes, added